security/azure/restrict_rdp_internet/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
```

Comment:
Looks like you accidentally branched this off of an old version of your other project instead of off of master.

Either remove the RDP files from this branch, or update them to match what was merged in the other branch.